src/software/ai/hl/stp/tactic/crease_defender/crease_defender_fsm.h

```diff
@@ -93,6 +127,15 @@ struct CreaseDefenderFSM
     static std::optional<Point> findDefenseAreaIntersection(
         const Field& field, const Ray& ray, double robot_obstacle_inflation_factor);
 
-   private:
+    /**
```

Comment:
still should be `private:`

Reply:
Oh i see why you made it public. The class members should still be `private` though

Reply:
Oh yeah, this part is a bit deceptive, I just moved the private up. These functions are actually private, which I suppose means theres no point making them static.